tasks/2017/spread-the-word.md

```diff
@@ -0,0 +1,83 @@
+# GCI Tasks: Spread The Word
+
+## Background
+
+Upvotes and reviews make a big difference in the public perception of projects
+like Zulip.
+
+## Task description
+
+In this task, we will ask you to rate and review Zulip on some technology sites.
+You're free to decide whether or not to do each individual task, but we do ask
+that you do at least five of the social media marketing tasks in order to get
+credit for the overall task. We expect completing all the tasks will take about
+90 minutes.
+
+Start by opening up a text file. You will be putting links and screenshots in
+this file in order to prove that you have completed parts of the task. We'll
+call this file the submission file.
+
+### Star us on GitHub.
+
+There are four main repositories:
+[server/web](https://github.com/zulip/zulip),
+[mobile](https://github.com/zulip/zulip-mobile),
+[desktop](https://github.com/zulip/zulip-electron), and [Python
+API](https://github.com/zulip/python-zulip-api). For each repository, star the
+repository, and then write down the number of stars that repository has in
```

Comment:
Maybe add something like "at the time of starring" or something to communicate that you don't have to keep updating this number.